vcf-dumper/README.md

```diff
@@ -0,0 +1,122 @@
+
+This command line interface includes the following parameters:
+* **species**: species the data are associated to
+* **database**: name of the database to extract data from
+* **outdir**: output directory
+* **studies**: comma separated list of studies to query
+* **files**: comma separated list of files to query (each study in EVA can be composed by one or many files, as described [here](https://github.com/EBIvariation/eva-pipeline/wiki/MongoDB-schema#files)
+
+A succesfull command execution will produce a VCF file in the output directory.
+
+#### Querying the test data
+For querying the test databases we need some values for the mandatory params:
+* species
+* studies
+* files (for CLI queries)
+* regions
+
+There are three databases in the test data: `eva_hsapiens_test`, `eva_btaurus_umd32_test` and `eva_oaries_oarv31_test`. `hsapiens_test`,  `btaurus_umd32_test` and `oaries_oarv31_test` would be valid values for the species parameter then. 
+
+For getting the files and studies ids in one database, we can query the collection *files* using the Mongo shell:
+```
+> use eva_hsapiens_test
+switched to db eva_hsapiens_test
+> db.files.find({},{sid:1, fid:1})
+{ "_id" : ObjectId("563cd52b9f44e8e80ea35828"), "fid" : "6", "sid" : "7" }
+{ "_id" : ObjectId("563cd5439f44e8e80ea35829"), "fid" : "5", "sid" : "8" }
+>
+```
+We can see in the query result that the study with id *7* is associated with the file with id *6*, and the study *8* with the file *5*. 
+
+For the regions filter, we need to know which chromosomes have variants in the database and where (position of the variants in the chromosome). As described in the [MongoDB schema wiki](https://github.com/EBIvariation/eva-pipeline/wiki/MongoDB-schema), each variant has *chr* (chromosome) and *start* (position in the chromosome where the variant start). We can figure out executing some queries in the *variants* collection:
+```
+> db.variants.distinct("chr", {"files.sid":"7"})
+[ "20", "22" ]
+> db.variants.find({"chr":"20", "files.sid":"7"},{"start":1}).sort({"start":1}).limit(1)
+{ "_id" : "20_60343_G_A", "start" : 60343 }
+> db.variants.find({"chr":"20", "files.sid":"7"},{"start":1}).sort({"start":-1}).limit(1)
+{ "_id" : "20_71822_C_G", "start" : 71822 }
+> db.variants.find({"chr":"22", "files.sid":"7"},{"start":1}).sort({"start":1}).limit(1)
+{ "_id" : "22_16050075_A_G", "start" : 16050075 }
+> db.variants.find({"chr":"22", "files.sid":"7"},{"start":1}).sort({"start":-1}).limit(1)
+{ "_id" : "22_16110950_G_T", "start" : 16110950 }
+```
+We can see that the study *7* has variants in the chromosomes *20* (between position 60343 to 71822) and *22* (between 16060075 and 16110950). 
+
+Now we have enough information for run some queries:
```

Comment:
Either is fine, personally I slightly prefer "for running" in this instance, but it doesn't make much difference.